int64 object_size
int64 human_action
int64 human_expression
---
float64 p_happy
float64 p_sad
float64 p_neutral
